packages/odyssey-storybook/src/components/odyssey-mui/Radio/Radio.stories.tsx

```diff
@@ -58,7 +58,7 @@ const selectRadio = async (
   await step("select the radio button", async () => {
     const canvas = within(canvasElement);
     const radio = canvas.getByRole("radio") as HTMLInputElement;
-    radio && (await userEvent.click(radio));
+    radio && userEvent.click(radio);
```

Comment:
Can we write this as `if (radio) { userEvent.click(radio) }`? I say that because we don't have an `&&` pattern like this without an explicit `if` at Okta.